apps/framework-cli/src/framework/core/infrastructure/view.rs

```diff
@@ -61,6 +64,20 @@ impl View {
     }
 }
 
+impl DataLineage for View {
```

Comment:
why are we adding stuff to this type anyway? it's subsumed by the more general `SqlResource`

Reply:
Yeah we probably should remove the `View` type altogether. I added it for completness

Reply:
I am leaving it for now but we should clean that up in a subsequent PR